python/pyspark/shell.py

```diff
@@ -45,11 +45,30 @@
         if parent_dir in sys.path:
             sys.path.remove(parent_dir)
 
-
```

Comment:
I didn't want to point this out but since it needs some changes .. I would revert this change back

Reply:
Done